addons/safemode/functions/fnc_playChangeFiremodeSound.sqf

```diff
@@ -16,12 +32,13 @@ if ({(toLower (_sound select 0) find _x == (count toArray (_sound select 0) - co
 };
 
 // add default volume, pitch and distance
-if (count _sound < 2) then {_sound pushBack 1};
-if (count _sound < 3) then {_sound pushBack 1};
-if (count _sound < 4) then {_sound pushBack 0};
+_soundCount = count _sound;
+if (_soundCount < 2) then {_sound pushBack 1};
+if (_soundCount < 3) then {_sound pushBack 1};
+if (_soundCount < 4) then {_sound pushBack 0};
```

Comment:
This might be true, doesn't change the outcome of this chain of conditions though, considering that if any of them are hit, all of the following are thus logically also executing.
Example:
`count _sound` is 2. meaning L36 doesn't hit, but L37 hits. It increases `count _sound` by 1, bringing it to 3, thus L38 also hits.